with-snowflake/README.md

```diff
@@ -0,0 +1,315 @@
+# StepZen Example: `with-snowflake`
+
+## Introduction
+
+This project builds a GraphQL API connected to your Snowflake warehouse using StepZen.
+We will use the `@dbquery` directive to generate our GraphQL schema.
+
+## Getting Started
+
+You'll need to create a [StepZen account](https://stepzen.com/request-invite) first. Once you've got that set up, [git clone](https://www.atlassian.com/git/tutorials/setting-up-a-repository/git-clone) this repository onto your machine and open the working directory:
+
+```bash
+git clone https://github.com/stepzen-dev/examples.git
+cd examples/with-snowflake
+```
+
+Before you can run this example, you need to have create a Snowflake warehouse following
+[Snowflake in 20 minutes](https://docs.snowflake.com/en/user-guide/getting-started-tutorial.html).
+
+## Snowflake in 20 minutes
+
+Follow the Snowflake getting started tutorial to create and populate the `EMP_BASIC` table in a warehouse.
+
+ * Snowflake in 20 minutes - https://docs.snowflake.com/en/user-guide/getting-started-tutorial.html
+
+Copy the file `sample.config.yaml` to a new `config.yaml` file:
+
+```bash
+cp sample.config.yaml config.yaml
+```
+
+Modify the contents of `config.yaml` to replace:
+
+  * `{{username}}` with your Snowflake user name
+  * `{{password}}` with your Snowflake password
+  * `{{account_identifier}}` with your Snowflake account identifier. The account identifier depends upon your Snowflake account edition. For information, see the [Snowflake documentation](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html).
+
+For example, your `config.yaml` should be similar to (depending on your chosen deployment cloud):
+
+```yaml
+# config.yaml
+
+configurationset:
+  - configuration:
+      name: snowy
+      dsn: 'jsmith:mypassword@ac79144.us-central1.gcp/SF_TUTS/PUBLIC?warehouse=SF_TUTS_WH'
+```
+
+## Run StepZen
+
+Open your terminal and [install the StepZen CLI](https://stepzen.com/docs/quick-start/install-and-setup):
+
+```bash
+npm install -g stepzen
+```
+
+You need to login here using the command:
+
+```bash
+stepzen login
+```
+
+After you've installed the CLI and logged in, run:
+
+```bash
+stepzen deploy
+```
+
+This creates a fully deployed managed GraphQL endpoint running in the cloud.
```

Comment:
hint to take note of the URL and show form here ... "you'll want this later to issue requests"